src/components/duelling-picklist/picklist-group/picklist-group.component.js

```diff
@@ -7,22 +13,79 @@ import {
   StyledPicklistGroup,
   StyledGroupButton,
 } from "./picklist-group.style";
+import FocusContext from "../duelling-picklist.context";
 import Events from "../../../utils/helpers/events/events";
 
 const PicklistGroup = React.forwardRef(
-  ({ title, children, type, onChange, ...rest }, ref) => {
+  (
+    { title, children, type, onChange, index, listIndex, isLastGroup, ...rest },
+    ref
+  ) => {
+    const { setElementToFocus, elementToFocus } = useContext(FocusContext);
     const [highlighted, setHighlighted] = useState(false);
-    const handleClick = useCallback(() => onChange(), [onChange]);
+
+    const filteredChildren = React.Children.toArray(children).filter(
+      (child) => child
```

Comment:
what this function does? Is there anything that is not a child inside?

Reply:
It's to filter out any null/ falsy children, I could have written it as `filter(Boolean)` tbh but it does the same thing under the hood

Reply:
Actually looking into it you're right, `toArray` will filter out any no truthy children so `filter` isn't needed